add-comment.html

```diff
@@ -51,12 +63,25 @@
           type: Boolean,
           value: false
         },
-        project: Object
+        project: Object,
+        _loading: {
+          type: Boolean,
+          value: false
+        }
       },
       _addComment: function () {
         if (this.comment.replace(/\s/g, '') !== '') {
-          this.fire('add-comment', this.comment);
-          this._hide();
+
+          var stopLoading = function (user) {
+            this._loading = false;
+            this._hide();
+          }.bind(this);
```

Comment:
Next time, you can make this into a `const` and use arrow syntax to avoid the `bind`:
```
const stopLoading = () => {
  this._loading = false;
  this._hide();
}
```